### PR TITLE
Creating tooltips

### PR DESCRIPTION
**Pull request by phulsechinmay**
The links for places found by the API now have tooltips shows when hovered. The tooltips currently show the pictures, name and the link to the wiki of the place.

### CODE REVIEW

File: display.js [hunk 1]
```diff
@@ -49,21 +49,22 @@ function createExtracts(results) {
 function activatePage() {
     var spinner = createSpinner();
     RESULTSBOX.appendChild(spinner);
-    return getLocations().then(
+    getLocations().then(
```
Comment (josiahcoad) [inline display.js]: the return has to be here so that it returns a promise
```suggestion
    return getLocations().then(
```

File: display.js [hunk 2]
```diff
@@ -49,21 +49,22 @@ function createExtracts(results) {
...
-                results.forEach(x =>
-                    searchAndReplace(
-                        document.body,
-                        x.spot,
-                        x.lod.wikipedia,
-                        createExtract(x)
-                    )
-                );
+                results.forEach(x => {
+                    var dataDict = {
```
Comment (josiahcoad) [inline display.js]: might be better to not make variable name and just pass the dict itself?

File: display.js [hunk 3]
```diff
@@ -90,7 +91,7 @@ chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
             () => sendResponse({
                 message: "FAILED"
             })
-        )
+        );
```
Comment (josiahcoad) [inline display.js]: we should agree on semicolons or not then find a way to enforce it. what do you think?

File: display.js (hunk 2 above)
Comment (phulsechinmay) [reply display.js]: I have the dataDict there so we can pass only the stuff we need, not everything from x

File: manifest.json [hunk 4]
```diff
@@ -5,14 +5,13 @@
...
-        "css": ["main.css"],
-        "js": ["jquery-3.3.1.min.js", "search.js", "logic.js", "display.js"],
+        "css": ["includes/css/bootstrap.min.css", "includes/css/tippy-light-theme.css", "main.css", "tooltip.css"],
+        "js": ["includes/js/jquery-3.3.1.min.js", "includes/js/bootstrap.min.js", "search.js", "logic.js", "display.js", "includes/js/tippy.all.min.js", "tooltip.js"],
         "matches": ["<all_urls>"]
     }],
     "browser_action": {
         "default_icon": "tab-icon.png",
-        "default_title": "SeeTheWorld",
-        "default_popup": "extension.html"
```
Comment (josiahcoad) [inline manifest.json]: a reason why this was deleted?

File: display.js (hunk 3 above)
Comment (phulsechinmay) [reply display.js]: Yup, weirdest thing about javascript and python. I am fine with either, what do you think? I tried to find if you were using either but couldn't find a specific style

File: manifest.json (hunk 4 above)
Comment (phulsechinmay) [reply manifest.json]: Nope, will revert it

File: tooltip.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+function InitializeTooltip(data, linkClass) {
+    console.log(data.image)
```
Comment (josiahcoad) [inline tooltip.js]: maybe not pushing console.log in code to master might be good going forward
```suggestion

```

Comment (phulsechinmay) [reply tooltip.js]: Yup, will remember that from now

File: manifest.json (hunk 4 above)
Comment (phulsechinmay) [reply manifest.json]:
```suggestion
        "default_popup": "extension.html"
```

File: display.js [hunk 6]
```diff
@@ -56,14 +56,15 @@ function activatePage() {
...
-                    )
-                );
+                results.forEach(x => {
+                    var dataDict = {
+                        search: x.spot,
+                        link: x.lod.wikipedia,
+                        image: x.image.thumbnail,
+                        summary: x.abstract
+                    };
+                    searchAndReplaceWithTooltip(document.body, dataDict);
```
Comment (josiahcoad) [inline display.js]: what about this: 
```suggestion
                    searchAndReplaceWithTooltip(document.body, {
                                            search: x.spot,
                                            link: x.lod.wikipedia,
                                            image: x.image.thumbnail,
                                            summary: x.abstract
                   });
```

Comment (phulsechinmay) [reply display.js]: Yup, that works

File: display.js (hunk 3 above)
Comment (josiahcoad) [reply display.js]: lol yeah, cause I didn't decide on one. haha. I could go either way. I did semi's in the past but think I'm leaning towards none.

Comment (phulsechinmay) [reply display.js]: None it is

Comment (josiahcoad) [reply display.js]: Ill look in sc more later though and remove them / add some enforcing contraints on another PR

File: display.js (hunk 2 above)
Comment (josiahcoad) [reply display.js]: now that we changed the below, we can remove this variable?

File: display.js (hunk 3 above)
Comment (phulsechinmay) [reply display.js]: Awesome

File: display.js (hunk 2 above)
Comment (phulsechinmay) [reply display.js]: Yeah, go ahead

**Pull request merged**